happi/tests/conftest.py

```diff
@@ -216,6 +216,11 @@ def getProposalDetailsForRun(self, run_no, proposal):
                 'pcdssetup-ai-1-name': 'irLed',
                 'pcdssetup-ai-1-purpose': 'IR LED',
                 'pcdssetup-ai-1-pvbase': 'MFX:USR:ai1:0',
+                'pcdssetup-motors-11-purpose': 'Von Hamos vertical',
+                'pcdssetup-motors-11-stageidentity': 'Beckhoff',
+                'pcdssetup-motors-11-location': 'XPP goniometer',
+                'pcdssetup-motors-11-pvbase': 'HXX:VON_HAMOS:MMS:01',
+                'pcdssetup-motors-11-name': 'vh_y_DONOTUSE',
```

Comment:
I think we can strip the DONOTUSE from the test here, Silke put this into the questionnaire because the loading didn't work and she didn't want anyone to accidentally use the broken object in hutch python

Reply:
took it out in 6c2d501 - thank you :)